src/routes/index.js

```diff
@@ -6,10 +6,14 @@
 import Router from '@koa/router';
 import { dockerFlowRoutes } from './dockerflow';
 import { publishRoutes } from './publish';
+import { versioning } from '../middlewares';
 
 export function routes() {
   const router = new Router();
   router.use(dockerFlowRoutes().routes());
+
+  // Versioning applies only to API routes, that's why we specify it here.
+  router.use(versioning('1.0'));
```

Comment:
Initially I inserted `versioning` in the `publishRoutes` file, but I thought it was too misleading, because it wasn't applying only to that file.
To make that obvious I moved it here.

Open question: 1.0 or 1 ? I think I'd rather use 1 for simplicity, what do you think?

Reply:
(I may find a better way to do it later)